scripts/train/train.py

```diff
@@ -466,15 +469,31 @@ def main(cfg: DictConfig) -> Trainer:
     ## Evaluation
     print('Building eval loader...')
     evaluators = []
-    eval_loader = None
+    eval_loaders = []
     if eval_loader_config is not None:
-        eval_dataloader = build_dataloader(eval_loader_config, tokenizer,
-                                           device_eval_batch_size)
-        eval_loader = Evaluator(
-            label='eval',
-            dataloader=eval_dataloader,
-            metric_names=[],  # we will add these after model is created
-        )
+        if isinstance(eval_loader_config, ListConfig):
+            for eval_config in eval_loader_config:
+                if eval_config.label is None:
+                    raise ValueError(
+                        'When specifying multiple evaluation datasets, each one must include the \
+                            `label` attribute.')
+                eval_dataloader = build_dataloader(eval_config, tokenizer,
+                                                   device_eval_batch_size)
+                eval_loader = Evaluator(
+                    label=f'eval/{eval_config.label}',
+                    dataloader=eval_dataloader,
+                    metric_names=[],  # we will add these after model is created
+                )
+                eval_loaders.append(eval_loader)
+        else:
+            eval_dataloader = build_dataloader(eval_loader_config, tokenizer,
+                                               device_eval_batch_size)
+            eval_loader = Evaluator(
+                label='eval',
+                dataloader=eval_dataloader,
+                metric_names=[],  # we will add these after model is created
+            )
+            eval_loaders.append(eval_loader)
```

Comment:
nit:
Suggested refactor:
```
is_multi_eval = isinstance(eval_loader_config, ListConfig)
eval_configs = eval_loader_config if is_multi_eval else [eval_loader_config]
eval_loaders = []
for eval_config in eval_configs:
    eval_dataloader = build_dataloader(...)
    eval_loader = Evaluator(
       label=f'eval/{eval_config.label}' if is_multi_eval else 'eval'
    )
   eval_loaders.append(eval_loader)
```

This makes it a little clearer that it's really just the label that is conditional when creating our Evaluator objects

Reply:
done, ty for the whole refactor!